### ISSUE TITLE
Cropping in netlify

### ISSUE THREAD

**Issue by emiliorizzo** [rbuchberger/jekyll_picture_tag#252]
Hi, I'm using cropping in 'attention' mode. It works fine in my local docker, but not in netlify. It crops the images but not to 'attention' area. Can you help me to solve that? Thanks!

**Comment by rbuchberger** [comment]
That probably has to do with the old version of vips that jekyll has in their build images. I'm not really sure about a good way to fix that, other than to use a different hosting option. Sorry I don't have a better answer for you; deploying with JPT can be painful sometimes.

I didn't know that was happening though, maybe we should add a warning to the docs about it.

**Comment by emiliorizzo** [comment]
I wrote to you first, but I'll try to ask for support in netlify. If I
found a solution, I'll let you know . Thanks for your answer,  JTP is very
useful! Thaks a lot.

**Comment by rbuchberger** [comment]
Hah, I feel bad because you won't be the first person bothering netlify support about my plugin. Hopefully they can help :)

**Comment by emiliorizzo** [comment]
Yes, I know. I've been reading about it. But I think the issues are related to netlify's images than your plugin :)
